src/tsickle.ts

```diff
@@ -609,8 +609,12 @@ class Annotator extends ClosureRewriter {
           this.emit(` {${exportedSymbols.map(e => unescapeName(e.name)).join(',')}}`);
         } else {
           if (exportDecl.exportClause) {
-            exportedSymbols =
-                exportDecl.exportClause.elements.map(e => typeChecker.getSymbolAtLocation(e.name));
+            exportedSymbols = exportDecl.exportClause.elements.map(e => {
+              return {
+                name: getIdentifierText(e.name),
```

Comment:
Ohh I see below it's about renaming exports.  Maybe dump a comment here about how/why sym.name and e.name will be different.